drivers/shared/src/get_parameters.c

```diff
@@ -563,3 +563,417 @@ get_parameters(FILE *paramfile)
         fgets(cmdstr, MAXSTRING, paramfile);
     }
 }
+
+/******************************************************************************
+ * @brief    Validate VIC model parameter values
+ *****************************************************************************/
+void
+validate_parameters()
+{
+
+    extern parameters_struct param;
+
+    // Validate Parameters
+    // Lapse Rate
+    if (param.LAPSE_RATE > 0.) {
```

Comment:
Is this an error or a warning. What if someone wants to do some experiment with inversions? Maybe leave it as error so people have to explicitly change it if they want to do that?